test/potluck_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'pry'
+require './lib/dish.rb'
+require './lib/potluck.rb'
+
+class PotluckTest < Minitest::Test
+  def test_dish_exists
+    potluck = Potluck.new("7-13-18")
+
+    assert_instance_of Potluck, potluck
+  end
+
+  def test_potluck_date
+    potluck= Potluck.new("7-13-18")
+
+    assert_equal "7-13-18", potluck.date
+  end
+
+  def test_if_dishes_default_empty_array
+    potluck = Potluck.new("7-13-18")
+
+    assert_equal [], potluck.dishes
+  end
+
+  def test_if_cous_added
```

Comment:
This should be testing that a dish can be added to the Potluck object.  it should look something like this:
```ruby
def test_dishes_can_be_added_to_potluck
  potluck = Potluck.new
  couscous_salad = Dish.new("Couscous Salad", :appetizer)
  potluck.add_dish(couscous_salad)
  assert_equal couscous_salad, potluck.dishes[0]
end
```